### ISSUE TITLE
Issues on Ubuntu

### ISSUE THREAD

**Issue by Buom01** [GrangerHub/Stellar-Prey#5]
Hi,
I don't know if you known that `./sp-tools.sh run client` doesn't work on Ubuntu after building it as documented in the `README.md`: https://pastebin.com/YFdggUDk

Error:
`./sp-tools.sh: line 715: /home/osboxes/Documents/Stellar-Prey/basepath/bin/unix/StellarPreyTest.AMD64: No such file or directory`

Debug informations:
https://pastebin.com/NkhuisvZ

I think that's probably due to an unprinted/handled error

Can I know which OS do you use to build it successfully ? I don't want to debug that but instead concentrating on assets and some other content

**Comment by cengique** [comment]
I also couldn't run it on Arch/Manjaro or Gentoo. I think it may be currently broken.

**Comment by TheDushan** [comment]
I really have no idea what @dGr8LookinSparky and co did.
With my repo and source code, everything is working fine.

**Comment by TheDushan** [comment]
> CMake Error at /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
>   Could NOT find OpenAL (missing: OPENAL_LIBRARY OPENAL_INCLUDE_DIR)

**Comment by Buom01** [comment]
Hi,
I have some news.

Even with all dependencies met, a part of the project compilation doesn't seem to launch: https://pastebin.com/3FFSbbiF
I suspect `sp-tools.sh` to not interact as expected with `./source/game-logic`, mainly with `cmake`. I didn't know cmake too much, but when the `zip` command throw error it affirms that some expected files are missing.

Maybe I did anything wrong a second time ?

Edit:
I just tested compiling it into the container I did. `./basepath/bin/unix/StellarPreyTest.AMD64` just pop out ... (even with the ending error).
It doesn't mean that all is clear: as it seem to run in the VM (and exit as there are no display) but it still found some small instabilities/issues. (Like bad engine library directoty  looking on fedora which is cwd and not script path, and other things normal for a WIP)
